test/test.jwt.ts

```diff
@@ -81,12 +81,17 @@ describe('JWT auth client', () => {
           ['http://bar', 'http://foo'], 'bar@subjectaccount.com');
 
       createGTokenMock();
-      jwt.authorize(() => {
+      jwt.authorize((err, creds) => {
+        assert.equal(err, null);
+        assert.notEqual(creds, null);
         assert.equal('foo@serviceaccount.com', jwt.gtoken.iss);
         assert.equal(PEM_PATH, jwt.gtoken.keyFile);
         assert.equal(['http://bar', 'http://foo'].join(' '), jwt.gtoken.scope);
         assert.equal('bar@subjectaccount.com', jwt.gtoken.sub);
         assert.equal('initial-access-token', jwt.credentials.access_token);
+        assert.equal(creds!.access_token, jwt.credentials.access_token);
```

Comment:
Wow!! I've never seen this before

Reply:
I hadn't either!  @ofrobots to the rescue.

Reply:
I mean csharp has it like `creds?.access_token` so I guess I shouldn't be surprised, but still nice.